# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/util-linux/no_tqueue.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./sys-utils/cytune.c.orig	Mon Apr 22 18:51:56 2002
+++ ./sys-utils/cytune.c	Mon Apr 22 18:52:14 2002
@@ -54,7 +54,7 @@
 #include <termios.h>
 
 #include "../defines.h"		/* for NEED_tqueue_h */
-#ifdef NEED_tqueue_h
+#ifdef NEED_tqueue_h_disabled
 #include <linux/tqueue.h>	/* required for old kernels (for struct tq_struct) */
 				/* compilation errors on other kernels */
 #endif
